modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -217,6 +217,8 @@ in one place, while also enabling features like schema evolution.
 
 Redpanda employs a unique strategy to help ensure the accuracy of retention operations. In this strategy, closed segments are only eligible for deletion when the age of all messages in the segment exceeds a configured threshold. However, when a producer sends a message to a topic, the timestamp set by the producer may not accurately reflect the time the message reaches the broker. To address this time skew, each time a producer sends a message to a topic, Redpanda records the broker's system date and time in the `broker_timestamp` property of the message. This property helps maintain accurate retention policies, even when the message's creation timestamp deviates from the broker's time.
 
+NOTE: Clock synchronization should be monitored by the server owner, as Redpanda does not monitor clock synchronization. While Redpanda does not rely on clocks for correctness, if  you are using LogAppendTime, server clocks may affect the time your application sees.
```

Comment:
```suggestion
NOTE: Clock synchronization should be monitored by the server owner, as Redpanda does not monitor clock synchronization. While Redpanda does not rely on clocks for correctness, if you are using `LogAppendTime`, server clocks may affect the time your application sees.
```

Reply:
Thanks @Deflaimun Added the parenthetical here, but think it doesn't hurt to keep in the linked topic as well.